media-api/app/model/Collection.scala

```diff
@@ -0,0 +1,50 @@
+package model
+
+import org.joda.time.DateTime
+
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+
+case class Collection(path: List[String], paradata: Paradata)
+object Collection {
+  implicit def formats: Format[Collection] = Json.format[Collection]
+}
+
+case class Paradata(who: String, when: DateTime)
```

Comment:
I'm trying to think of something generic that we could the apply to, say the application pf these collections to images, labels, uploading, editing metadata etc.